backend/src/Designer/Filters/ProblemDetailsExtensionsCodes.cs

```diff
@@ -0,0 +1,7 @@
+namespace Altinn.Studio.Designer.Filters;
+
+public static class ProblemDetailsExtensionsCodes
+{
+    public const string ErrorCode = "errorCode";
```

Comment:
Does it make sense giving this a more descriptive name? If we are going to expand this class with more errorCodes at a later point